### ISSUE TITLE
Find a pattern and document for other uniffied application-services in the megazord

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/application-services#3968]
Existing hand-crafted FFIs for existing components are being replaced by uniffied FFIs.

That work may have a better way of integrating than the one already found for Nimbus. Or vice versa. This issue allows for the work of changing the nimbus integration, and documenting the Better Way™.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/SYNC-2130)
┆Story Points: 3
┆Epic: Integrate Nimbus-SDK into iOS
┆Sprint: SYNC - 2021-04-09

**Comment by data-sync-user** [comment]
➤ Ryan Kelly commented:

[~jhugman@mozilla.com] do you think we're in a good place with this pattern now, and the documentation thereof? I know we've got [1] for docs in uniffi, and your example iOS app in there. Do you think we also need some megazord-specific instructions that live in the appservices repo?

[1] https://github.com/mozilla/uniffi-rs/pull/405

**Comment by data-sync-user** [comment]
➤ James Hugman commented:

I've added one crate to the iOS megazord, you've added a second. I think we have enough for a putting together a single page document.

My suspicion is that we've done the hard part by adding the first two and that any document should be fairly small (a few Xcode screenshots we've already talked about) to document the marginal steps of adding the next one, rather that starting from scratch.

In the context of https://github.com/mozilla/uniffi-rs/pull/416 and experiments in getting rid of the UDL, I'd counsel that we don't spend a lot of time on  it given a lot of it will change in the relatively near future, but we should'nt use that as an argument not to do anything.
